uv/uv_functions.php

```diff
@@ -0,0 +1,1336 @@
+ *
+ * @param resource $handle uv resource handle (uv rwlock).
+ */
+function uv_rwlock_rdlock(resource $handle): void {}
+
+/**
+ * @param resource $handle
+ *
+ * @return bool
+ */
+function uv_rwlock_tryrdlock(resource $handle): bool {}
+
+/**
+ * Unlock read lock.
+ *
+ * @param resource $handle uv resource handle (uv rwlock)
+ *
+ * @return void
+ */
+function uv_rwlock_rdunlock(resource $handle): void {}
+
+/**
+ * Set write lock.
+ *
+ * @param resource $handle uv resource handle (uv rwlock).
+ *
+ * @return void
+ */
+function uv_rwlock_wrlock(resource $handle): void {}
+
+/**
+ * @param resource $handle
+ */
+function uv_rwlock_trywrlock(resource $handle): void {}
+
+/**
+ * Unlock write lock.
+ *
+ * @param resource $handle uv resource handle (uv rwlock).
+ */
+function uv_rwlock_wrunlock(resource $handle): void {}
+
+/**
+ * Initialize mutex resource.
+ *
+ * @return resource uv mutex resource
+ */
+function uv_mutex_init(): resource {}
+
+/**
+ * Lock mutex.
+ *
+ * @param resource $lock uv resource handle (uv mutex).
+ *
+ * @return void
+ */
+function uv_mutex_lock(resource $lock): void {}
+
+/**
+ * @param resource $lock
+ *
+ * @return bool
+ */
+function uv_mutex_trylock(resource $lock): bool {}
+
+/**
+ * Initialize semaphore resource.
+ *
+ * @param int $value
+ * @return resource
+ */
+function uv_sem_init(int $value): resource {}
+
+/**
+ * Post semaphore.
+ *
+ * @param resource $sem uv resource handle (uv sem).
+ *
+ * @return void
+ */
+function uv_sem_post(resource $sem): void {}
+
+/**
+ * @param resource $sem
+ *
+ * @return void
+ */
+function uv_sem_wait(resource $sem): void {}
+
+/**
+ * @param resource $sem
+ *
+ * @return void
+ */
+function uv_sem_trywait(resource $sem): void {}
+
+/**
+ * Initialize prepare resource.
+ *
+ * @param resource $loop uv loop handle.
+ *
+ * @return resource
+ */
+function uv_prepare_init(resource $loop): resource {}
+
+/**
+ * Setup prepare loop callback. (pre loop callback)
+ *
+ * @param resource $handle uv resource handle (prepare)
+ * @param callable $callback this callback parameter expects (resource $prepare, long $status).
+ *
+ * @return void
+ */
+function uv_prepare_start(resource $handle, callable $callback): void {}
+
+/**
+ * Stop prepare callback.
+ *
+ * @param resource $handle uv resource handle (prepare).
+ *
+ * @return void
+ */
+function uv_prepare_stop(resource $handle): void {}
+
+/**
+ * Setup check resource.
+ *
+ * @param resource $loop uv loop handle
+ *
+ * @return resource
+ */
+function uv_check_init(resource $loop): resource {}
+
+/**
+ * Stats check loop callback. (after loop callback)
+ *
+ * @param resource $handle uv resource handle (check).
+ * @param callable $callback this callback parameter expects (resource $check, long $status).
+ *
+ * @return void
+ */
+function uv_check_start(resource $handle, callable $callback): void {}
+
+/**
+ * Stop check callback.
+ *
+ * @param resource $handle uv resource handle (check).
+ *
+ * @return void`
```

Comment:
Please change to `@return void` (remove the closing `)

Reply:
For a moment I was pretty impressed you managed to spot something I thought was a bit of dust. Then I realised the tests failed :joy:

Reply:
Ok fixed it, messed up rebase, fixed that as well